classes/rental.rb

```diff
@@ -3,11 +3,14 @@ class Rental
 
   def initialize(date, book, person)
     @date = date
-
+    @person = person
     @book = book
+
+    person.rental << self
     book.rentals << self
+  end
 
-    @person = person
-    person.rentals << self
```

Comment:
While trying to create a rental I got the error below:

![Screenshot 2023-11-10 at 11 40 33 AM](https://github.com/evansnyamekye/school-library/assets/79506105/0faa4327-444f-4cda-aec8-273556537571)


It would be nice to fix this issue.